automation/check-patch.sh

```diff
@@ -95,6 +95,8 @@ cleanup() {
     # Otherwise the environment cleanup code will clean them up as well as the
     # Docker data disks that we want to keep around.
     vagrant destroy
+    vagrant global-status --prune
+    ps aux | grep ruby | awk '{print $2}' | xargs kill -9
```

Comment:
can you put in a comment that provides some background for why this 'kill' of ruby pids is necessary